LICENSE.md

```diff
@@ -9,4 +9,4 @@ THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR
 CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
 LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
 NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
-CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
\ No newline at end of file
+CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
```

Comment:
Weird

Reply:
Do you think we should change the license ?

Reply:
No sorry, I meant the change is weird as nothing really changed

Reply:
yes you're right 🤔 
Maybe just prettier ?

Reply:
Yes maybe some hidden chars